yoti_python_sdk/dynamic_sharing_service/policy/source_constraint_builder.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from yoti_python_sdk.config import PASSPORT, PASS_CARD, NATIONAL_ID, DRIVING_LICENCE
+
+from .wanted_anchor_builder import WantedAnchorBuilder
+
+
+class SourceConstraintBuilder(object):
+    def __init__(self):
+        self.__soft_preference = False
+        self.__anchors = []
+
+    def with_soft_preference(self, value=True):
+        """
+        :param value: If true, this will be treated as a soft preference, otherwise
+                      this will be treated as a hard requirement
+        """
+        self.__soft_preference = value
+        return self
+
+    def with_anchor(self, anchor):
+        """
+        :param anchor: Add an anchor to the source constraint. You should prefer
```

Comment:
Not sure if we should say `You should prefer (to) use`, should we say `It is recommended to use the other helper methods instead of this one`?